.github/workflows/pr.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           ./mvnw ${MAVEN_ARGS} formatter:validate -Dconfigfile=$PWD/contributing/eclipse-google-style.xml -pl '!operator-framework-bom' --file pom.xml
           ./mvnw ${MAVEN_ARGS} impsort:check -pl '!operator-framework-bom' --file pom.xml
       - name: Run unit tests
-        run: ./mvnw ${MAVEN_ARGS} -B test --file pom.xml
+        run: ./mvnw ${MAVEN_ARGS} clean install --file pom.xml
```

Comment:
Why was this changed to to an install instead of simply testing?

Reply:
The bom was then not accessible from the project, if there is no install